modules/shortcodes/css/slideshow-shortcode.css

```diff
@@ -99,6 +99,7 @@ body div.slideshow-window * img {
 	-webkit-transition: 300ms opacity ease-out;
 	-moz-transition: 300ms opacity ease-out;
 	transition: 300ms opacity ease-out;
+	direction: ltr;
```

Comment:
Looks like `direction` was already here, in the line 98:

https://github.com/Automattic/jetpack/blob/6c2f99c36d0f5ec723e67a20311e91b120bde3c5/modules/shortcodes/css/slideshow-shortcode.css#L98-L102

There's `/* @noflip */` just above this CSS class, does that have something to do with original issue?

Reply:
Oh heh, I need new glasses I guess.:) Thanks for taking a look!

> There's /* @noflip */ just above this CSS class, does that have something to do with original issue? 

That `/* @noflip */` rule does not appear to be respected (`direction` gets flipped in the RTL file anyway), so that seems to be the culprit here indeed. 

I pushed a new commit with the correct notation (`/*rtl:ignore*/`) for the library we use now, and it seems to work! I believe I got tricked by a cached file earlier.